src/TrigScint/TrigScintQIEDigiProducer.cxx

```diff
@@ -83,8 +82,11 @@ void TrigScintQIEDigiProducer::produce(framework::Event& event) {
   }
 
   // To simulate multiple pulses coming at different times, SiPMS
-  float TrueEdep[stripsPerArray_];
-  Expo* ex[stripsPerArray_] = {0};
+  // Initialize with stripsPerArray_ zeros
+  std::vector<float> TrueEdep(stripsPerArray_, 0.);
+
+  // Initialize with stripsPerArray_ nullptrs
+  std::vector<Expo*> ex(stripsPerArray_, nullptr);
```

Comment:
Here's the change from using variable length arrays to as std::vector. The constructor being used is the one which produces N (`stripsPerArray_`) copies of an initial value